app/javascript/app/components/country-ndc-overview/country-ndc-overview-component.jsx

```diff
@@ -24,6 +24,11 @@ class CountryNdcOverview extends PureComponent {
         />
       );
     }
+    const hasCoverageSectorsShort =
```

Comment:
It seems strange to me to do this for one value and not the others. Seems hacky to me. Can we not address the selectors or the data to prevent rendering undefined?